app/components/Message.tsx

```
@@ -22,82 +19,83 @@ export const Message = ({
-            if (part.type === "text") {
-              return (
-                <MemoizedMarkdown
-                  key={`${message.id}-text`}
-                  id={message.id}
-                  content={part.text}
-                />
-              );
+          {message.parts.map((part, partIndex) => {
+            switch (part.type) {
+              case "text":
+                const partId = `${message.id}-text-${partIndex}`;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)